product-scenarios/1-manage-public-partner-private-apis/1.5-expose-api-to-the-app-developer/1.5.2-expose-api-to-subset-of-app-developer-restricted-by-roles/src/test/java/org/wso2/am/scenario/tests/rest/api/restrictedVisibility/RestApiVisibilityRestrictedByRolesTestCase.java

```diff
@@ -293,11 +280,10 @@ public void destroy() throws Exception {
                 PUBLISHER_CREATOR_USERNAME);
         apiPublisher.deleteAPI("APIVisibility_tagsDistinctRoles2", apiVersion,
                 PUBLISHER_CREATOR_USERNAME);
-//        todo uncomment when 1.5.2.10 is fixed
-//        apiPublisher.deleteAPI("APIVisibility_apiByTag1", apiVersion,
-//                PUBLISHER_CREATOR_USERNAME);
-//        apiPublisher.deleteAPI("APIVisibility_apiByTag2", apiVersion,
-//                PUBLISHER_CREATOR_USERNAME);
+        apiPublisher.deleteAPI("APIVisibility_apiByTag1", apiVersion,
+                PUBLISHER_CREATOR_USERNAME);
```

Comment:
Can't we add this in single line.

Reply:
Fixed with https://github.com/wso2/product-apim/pull/5248